elro/auth.py

```diff
@@ -76,6 +82,31 @@ def session(self) -> ElroConnectsCloudSessionCache | None:
         """Return the current session."""
         return self._session_cache
 
+    def _get_domain(self):
+        """Return the API main domain name address"""
+
+        sckt = socket.socket(socket.AF_INET,socket.SOCK_STREAM)
+        host="info.hekr.me"
+        port=91
+        sckt.connect((host,port))
+        message = "{\"action\":\"getAppDomain\"}\n"
```

Comment:
How is the domain determined. Is this based on the source address of the client?